src/sections/Welcome.js

```diff
@@ -0,0 +1,109 @@
+import { css } from "@emotion/core"
+import styled from "@emotion/styled"
+import React from "react"
+import Particles from "react-tsparticles"
+import Button from "../components/Button"
+
+function Welcome(props) {
+  const ParticlesContainer = styled.div`
+    width: 100vw;
+    height: 90vh;
```

Comment:
This way your particles will overlap with everything after the welcome section. You could use 90% here, then the particles will end 10% above the bottom of the welcome section